app/com/gu/floodgate/reindex/ProgressTrackerController.scala

```diff
@@ -18,7 +18,7 @@ object ProgressTrackerController {
 
 class ProgressTrackerController(ws: WSAPI, runningJobService: RunningJobService, jobHistoryService: JobHistoryService) extends Actor with ActorLogging with StrictLogging {
 
-  private var runningTrackers: Map[String, ActorRef] = Map.empty[String, ActorRef]
+  private var runningTrackers: Map[ContentSource, ActorRef] = Map.empty[ContentSource, ActorRef]
```

Comment:
Using the `ContentSource` itself as the key is potentially a little dangerous. e.g. if somebody updates the content source's description during a reindex, you will lose track of it in this map.

Maybe generate a unique ID based on the ID and the environment? e.g.

```scala
case class ContentSource(...) {
   def uniqueId: String = s"$id-$environment"
}